src/Microsoft.ServiceFabric.Services.Remoting/DeprecationMessage.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.ServiceFabric.Services.Remoting
+{
+    internal static class DeprecationMessage
+    {
+        public const string RemotingV1 = "Service Remoting V1 is deprecated. Please use Service Remoting V2 instead.";
```

Comment:
Could you start a PR to add `Deprecated/RemotingV1.md` to this folder: https://github.com/microsoft/service-fabric/tree/master/release_notes so that we can add the following link to this message string:
https://github.com/microsoft/service-fabric/blob/master/release_notes/deprecated/RemotingV1.md